src/main.rs

```diff
@@ -94,6 +96,23 @@ fn get_issue_tree_init_hashes(repo: &Repository, _: &clap::ArgMatches) -> i32 {
 }
 
 
+/// check-message subcommand implementation
+///
+fn check_message(matches: &clap::ArgMatches) -> i32 {
+    let reader: Box<Read> = match matches.value_of("filename") {
+        Some(filename)  => Box::from(try_or_1!(File::open(filename))),
+        None            => Box::from(io::stdin()),
+    };
+    BufReader::new(reader).lines()
+                          .map(|l| l.unwrap())
```

Comment:
The only way the `unwrap()` panics is on an IO error. It's valid to abort the program in such cases, IMO. We also could throw in an explicit `exit()` instead as an alternative.